src/components/modules/api/saver.ts

```diff
@@ -14,14 +14,16 @@ export default class SaverAPI extends Module {
    */
   public get methods(): Saver {
     return {
-      save: (): Promise<OutputData> => this.save(),
+      save: (sanitizeDisable = true): Promise<OutputData> => this.save(sanitizeDisable),
```

Comment:
I think default sanitizeDisable should be `false` for safety.

Reply:
yes you are right

Reply:
@hata6502 As I end up discussion https://github.com/codex-team/editor.js/issues/1280 sanitizing should better be focus on input and on server side, it is good however that a first stage of sanitizing on pasting, rendering. and on inserting blocks has been done but for large number of blocks this overload on client side gives this https://github.com/codex-team/editor.js/issues/1191 . However, this patch that imho has no security issue because we work not input but on output, readonly for block content .